driver/bpf/fillers.h

```diff
@@ -456,11 +456,11 @@ static __always_inline int bpf_parse_readv_writev_bufs(struct filler_data *data,
 
 #ifdef BPF_FORBIDS_ZERO_ACCESS
 				if (to_read)
-					if (bpf_probe_read(&data->buf[off & SCRATCH_SIZE_HALF],
+					if (bpf_probe_read(&data->buf[off & SCRATCH_SIZE_HALF - 1],
```

Comment:
Parentheses? Is this expression `(off & SCRATCH_SIZE_HALF) - 1` or `off & (SCRATCH_SIZE_HALF - 1)`?

Reply:
- has precedence over & and I confirm that the generated bpf bytecode is correct.

Reply:
Based on what you wrote above, is it possible that the intent of the code is `(off & SCRATCH_SIZE_HALF) - 1`?